projects/aca-folder-rules/src/lib/rule-details/conditions/rule-condition-fields.ts

```diff
@@ -23,7 +23,7 @@
  * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
  */
 
-export type RuleConditionFieldType = 'string' | 'number' | 'date' | 'type' | 'special';
+export type RuleConditionFieldType = 'string' | 'number' | 'date' | 'type' | 'special' | 'equals_comparator';
```

Comment:
"equals_comparator" isn't a type. Can we use `'special'` similar to how "category", "tag" and "aspect" do it. This  should hide the comparator field entirely (since equals is the only option"). See here how it is included as a comparator used for the `'special'` type (even though it is eventually hidden by the component): https://github.com/Alfresco/alfresco-content-app/blob/develop/projects/aca-folder-rules/src/lib/rule-details/conditions/rule-condition-comparators.ts#L40

Reply:
I have now used 'special' comparator instead of 'equals_comparator'.